docs/02.API-REFERENCE.md

```diff
@@ -9,6 +9,7 @@ Enum that contains the following elements:
  - JERRY_INIT_SHOW_REGEXP_OPCODES - dump regexp byte-code to log after compilation
  - JERRY_INIT_MEM_STATS - dump memory statistics
  - JERRY_INIT_MEM_STATS_SEPARATE - dump memory statistics and reset peak values after parse
+ - JERRY_INIT_LEXICAL_ENV - enable the lexical environment for the debugger
```

Comment:
This should be JERRY_INIT_DEBUGGER, and enables all features required by debugging. Otherwise this would be a too complex api.